src/settings.rs

```diff
@@ -58,9 +58,10 @@ pub struct Settings {
     #[serde(skip)]
     override_steam_api_key: Option<Arc<str>>,
     #[serde(skip)]
-    override_port: Option<u16>,
+    override_webui_port: Option<u16>,
     #[serde(skip)]
     override_steam_user: Option<SteamID>,
+    rcon_port: u16,
```

Comment:
I'd prefer to put this just below `external` instead to keep the overrides all at the bottom. It might be a good idea to also add an override for the rcon port if you want to, but no big deal.

Reply:
Done in [`0f89065` (#94)](https://github.com/MegaAntiCheat/client-backend/pull/94/commits/0f890655d2a189c3cbdedc99a267cbcbe4477bf1)